### PR TITLE
Update VulkanSDK to 1.1.121.2

### PR DESCRIPTION
**Pull request by emulationchannel**

### CODE REVIEW

**Comment by dracc** [comment]
Much better!
This could however be just one commit as it's such a small change code wise and the changes are depending on each other.
Please use `git rebase -i master` and `git push -f` on your current branch instead of closing this PR and opening a new one.

**Comment by emulationchannel** [comment]
ok

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can find us for developer discussion:
 - Linux with a modern C++17 friendly compiler
 - 64 bit
 - CMake v3.2+
-- OpenGL 4.5+ or Vulkan 1.1.92.1+
+- OpenGL 4.5+ or Vulkan 1.1.121.2+
```
Comment (exjam) [inline README.md]: There is no need to change the minimum version requirement in the README as decaf only uses features from vulkan making 1.1.92.1 the minimum version.

Comment (emulationchannel) [reply README.md]: ok

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ You can find us for developer discussion:
 - Linux with a modern C++17 friendly compiler
 - 64 bit
 - CMake v3.2+
-- OpenGL 4.5+ or Vulkan 1.1.121.2+
+- OpenGL 4.5+ or Vulkan  1.1.92.1+
```
Comment (dracc) [inline README.md]: An extra space slipped in. Please remove this commit and a7cfd86b07aa instead.

**Comment by dracc** [comment]
Nope, my suggested change has not been fixed.

**Comment by emulationchannel** [comment]
yes fixed reviewed exjam  Vulkan 1.1.92.1+

**Comment by dracc** [comment]
My review comment was regarding the space that slipped into your fourth commit (a3b0d6b) trying to revert the changes of a7cfd86 - you still have not fixed this.

**Pull request closed without merging**